github/pulls_reviews.go

```diff
@@ -59,6 +59,11 @@ func (r PullRequestReviewDismissalRequest) String() string {
 
 // ListReviews lists all reviews on the specified pull request.
 //
+// TODO: Follow up with GitHub support about an issue with this method's
+// returned error format and remove this comment once it's fixed.
+// Read mroe about it here -
+// https://github.com/google/go-github/pull/497#issuecomment-267810912
```

Comment:
There's a typo here, "mroe" should be "more".

Instead of linking to that PR with 100+ of comments, let's use a new issue to track this and point people at it. This will be much friendlier.

I've made https://github.com/google/go-github/issues/540 - please use it.